lib/graphiti/errors.rb

```diff
@@ -816,5 +816,34 @@ def message
 
     class ConflictRequest < InvalidRequest
     end
+
+    class FilterGroupInvalidRequirement < Base
+      def initialize(resource, valid_required_values)
+        @resource = resource
+        @valid_required_values = valid_required_values
+      end
+
+      def message
+        <<-MSG.gsub(/\s+/, " ").strip
+          The filter group required: value on resource #{@resource.class} must be one of the following:
```

Comment:
how about `@resource.jsonapi_type` so that it more closely matches the url param? This will provide users with actionable error messages once we wire these up to graphiti-rails error handling.

(This might be wrong if my comment on the validator class below is correct, as this is more a message for API devs, not users. Would still apply to the other error message below though)

Reply:
Yes this error is more for devs in the case of passing in incorrect values during dev.